src/TwigComponent/src/ComponentAttributes.php

```diff
@@ -146,4 +150,9 @@ public function add($stimulusDto): self
         // add the remaining attributes for values/classes
         return $clone->defaults($controllersAttributes);
     }
+
+    public function remove($key): void
+    {
+        unset($this->attributes[$key]);
+    }
```

Comment:
The other methods in this class are immuatable - so we should probably do the same thing here: clone and return a modified version of ourself. Then in the Twig magic, it'll need to be:

```
$context[\'attributes\'] = $context[\'attributes\']->remove(...)
```